contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -86,8 +86,6 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
         uint32 unclaimedTokens,
         address ipIdAddress
     ) external initializer {
-        if (ipIdAddress == address(0)) revert Errors.IpRoyaltyVault__ZeroIpId();
-
```

Comment:
We don't need this check?

Reply:
why removed this check?

Reply:
this check was removed because the deployment is made by RoyaltyPolicyLAP - during `initPolicy` which will revert upstream if ipId is zero

Reply:
please add it back to keep it safe

Reply:
https://github.com/storyprotocol/protocol-core-v1/pull/78/commits/a2a189032de54636a0fd88ca3d2276eb31ecfc75